.travis.yml

```diff
@@ -4,6 +4,7 @@ php:
     - 5.4
     - 5.5
     - 5.6
+    - nightly
```

Comment:
nightly makes no sense, beause in one year, nightly will not mean 7.0, and then 7.0 will be untested